l10n_it_declaration_of_intent/tests/test_declaration_of_intent.py

```diff
@@ -503,3 +503,22 @@ def test_action_register_payment(self):
         else:
             payments = action["domain"][0][2]
             self.assertTrue(len(payments) > 1)
+
+    def test_multiple_valid(self):
+        declaration_model = self.env["l10n_it_declaration_of_intent.declaration"].sudo()
+        post_used_amount2 = self.declaration2.used_amount
+        post_used_amount3 = self.declaration3.used_amount
+        self.assertAlmostEqual(post_used_amount2, 0.0, 2)
+        self.assertAlmostEqual(post_used_amount3, 0.0, 2)
+        records = declaration_model.get_valid(
+            type_d="out", partner_id=self.partner2.id, date=self.today_date
+        )
+        self.assertEqual(len(records), 2)
```

Comment:
```suggestion
        valid_declarations = declaration_model.get_valid(
            type_d="out", partner_id=self.partner2.id, date=self.today_date
        )
        self.assertEqual(valid_declarations, self.declaration2 | self.declaration3)
```
o almeno dare un nome più significativo invece di `records`.